apps/design/frontend/src/ballot_order_info_screen.tsx

```diff
@@ -0,0 +1,187 @@
+  MainHeader,
+  CheckboxButton,
+} from '@votingworks/ui';
+import { useParams } from 'react-router-dom';
+import { Id } from '@votingworks/types';
+import type { BallotOrderInfo } from '@votingworks/design-backend';
+import styled from 'styled-components';
+import { Form, FormActionsRow, InputGroup } from './layout';
+import { ElectionNavScreen } from './nav_screen';
+import { getElection, updateBallotOrderInfo } from './api';
+
+export const Annotation = styled.div`
+  margin-top: -1rem;
+`;
+
+function BallotOrderInfoForm({
+  electionId,
+  savedBallotOrderInfo,
+}: {
+  electionId: Id;
+  savedBallotOrderInfo: BallotOrderInfo;
+}): JSX.Element {
+  const [isEditing, setIsEditing] = useState(false);
+  const [ballotOrderInfo, setBallotOrderInfo] =
+    useState<BallotOrderInfo>(savedBallotOrderInfo);
+  const updateBallotOrderInfoMutation = updateBallotOrderInfo.useMutation();
+
+  function onSaveButtonPress() {
+    updateBallotOrderInfoMutation.mutate(
+      {
+        electionId,
+        ballotOrderInfo: { ...savedBallotOrderInfo, ...ballotOrderInfo },
+      },
+      { onSuccess: () => setIsEditing(false) }
+    );
+  }
+
+  return (
+    <Form>
+      <InputGroup label="Number of Absentee Ballots">
+        <input
+          type="text"
+          value={ballotOrderInfo.absenteeBallotCount ?? ''}
+          onChange={(e) =>
+            setBallotOrderInfo({
+              ...ballotOrderInfo,
+              absenteeBallotCount: e.target.value,
+            })
+          }
+          disabled={!isEditing}
+        />
+      </InputGroup>
+      <div>
+        <CheckboxButton
+          label="Score Absentee Ballots for Folding"
+          isChecked={Boolean(
+            ballotOrderInfo.shouldAbsenteeBallotsBeScoredForFolding
+          )}
+          onChange={(isChecked) =>
+            setBallotOrderInfo({
+              ...ballotOrderInfo,
+              shouldAbsenteeBallotsBeScoredForFolding: isChecked,
+            })
+          }
+          disabled={!isEditing}
+        />
+      </div>
+      <InputGroup label="Number of Polling Place Ballots">
+        <input
+          type="text"
+          value={ballotOrderInfo.precinctBallotCount ?? ''}
+          onChange={(e) =>
+            setBallotOrderInfo({
+              ...ballotOrderInfo,
+              precinctBallotCount: e.target.value,
+            })
+          }
+          disabled={!isEditing}
+        />
+      </InputGroup>
+      <InputGroup label="Paper Color for Polling Place Ballots">
+        <input
+          type="text"
+          value={ballotOrderInfo.precinctBallotColor ?? ''}
+          onChange={(e) =>
+            setBallotOrderInfo({
+              ...ballotOrderInfo,
+              precinctBallotColor: e.target.value,
+            })
+          }
+          disabled={!isEditing}
+        />
+      </InputGroup>
+      <Annotation>
+        Specify if for town or school ballots. If not specified, we’ll print on
+        white.
+      </Annotation>
```

Comment:
This annotation is of course specific to NH local elections - not worrying about generalizing this form right now